src/app/App.tsx

```diff
@@ -12,15 +11,13 @@ import './infrastructure/faro';
 export function App() {
   return (
     <QueryClientProvider client={queryClient}>
-      <GitHubContextProvider>
```

Comment:
Move to `SceneProfilesExplorer` because we need the current data source UID for build the VCS clients required by `GitHubContextProvider `.